met-web/src/components/common/index.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-import { Button as MuiButton, Paper as MuiPaper, Grid, CircularProgress } from '@mui/material';
+import { Button as MuiButton, Container, Grid, CircularProgress } from '@mui/material';
 import styled from '@emotion/styled';
 
 export const RoundedButton = styled(MuiButton)(() => ({
     borderRadius: '23px',
 }));
 
-export const MetBox = styled(MuiPaper)(() => ({
-    border: `1px solid #606060`,
+export const MetBox = styled(Container)(() => ({
```

Comment:
Why?
- Container adds padding left and right, while Paper does not. MetBox  was not used (in engagement form for example) with the intention of it behaving like a MUI Container.
- We would do better to avoid overwriting components in the common folder and create new ones if we needed them.